.coderabbit.yaml

```
@@ -0,0 +1,69 @@
+
+auto_review:
+  enabled: true                  # 자동 리뷰 활성화
+  drafts: false                  # 초안 PR 자동 리뷰 비활성화
+  ignore_title_keywords:         # 제목 키워드로 자동 리뷰 무시
+    - "[docs]"
+    - "[skip-review]"
+
+# 활성화된 도구 설정
+tools:
+  markdownlint:
+    enabled: true # 문서 검사 
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)